### PR TITLE
Always show welcome page

### PR DESCRIPTION
**Pull request by ableuler**
Make sure the content of the templateable welcome page is accessible to all users, also the users who are logged in and have already starred a project.

### CODE REVIEW

**Comment by ableuler** [comment]
@ciyer @rokroskar up on staging now!

**Comment by rokroskar** [comment]
tested, looks fine to me

File: src/landing/Landing.present.js [hunk 1]
```diff
@@ -203,16 +203,17 @@ class Home extends Component {
     let selected = this.props.ui.selected;
     let nav = null;
     const urlMap = this.props.urlMap;
-    let welcome = <AnonymousWelcome urlMap={urlMap} welcomePage={this.props.welcomePage} />;
+    let welcome;
```
Comment (ciyer) [inline src/landing/Landing.present.js]: I would prefer the following:
```
const anonymous = this.props.user != null && this.props.user.id != null;
const welcome = <Welcome welcomePage={this.props.welcomePage} anonymous={anonymous}/>;
if (anonymous) { 
  nav = [...]
} else {
  [...]
}
```

**Comment by ableuler** [comment]
@ciyer Good point. Adapted and thereby also used the `loggedIn` property already defined higher up in the tree.

**Pull request merged**